### DISCUSSION TITLE
Not Allow two GoogleDrive Conections

### DISCUSSION THREAD

**Discussion by naviges** [iterate-ch/cyberduck#14931] [Q&A] [answered]
Hi,

I try to add another GoogleDrive link, it do sucess bu after close and restart de app only show one GoogleDrive link, the others has disapear.

Thanks

**Comment by tatze96** [comment]
Did you use the *Open Connection* button to the top left or create a bookmark through the Cyberduck menu or respectively with the help of the "+" button in the lower-left corner of the Cyberduck bookmark view?

Do you get any error messages? If so, send us screenshots. Ensure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.

**Comment by wayo** [comment]
Good day! I would like to understand, I do not have an active field for editing in which you need to enter mail data. Is this how it should be, how it is intended or is it a glitch on my system?
![cd](https://github.com/iterate-ch/cyberduck/assets/183312/fb7c92e9-5619-4cbf-bd1c-971553b72be5)

**Reply by tatze96** [reply]
The fields of this protocol are greyed out purposely because you'll get redirected to an OAuth page within the web browser on the first connection of the bookmark. The credential fields will be filled in automatically once the authorization process is finished.

To connect to the drive, close the bookmark configuration window and double-click on the Google Drive entry within the bookmark list. You'll get redirected to a web browser page on which you can log in with your account credentials.

**Reply by ribasc** [reply]
I tried doing that, but instead of being redirected, it automatically logs me into the Google account that was already connected, even if I logged out of it first. What should I do?

**Reply by dkocher** [reply, marked as answer]
You will have to logout from your Google account in the web browser prior opening the new connection.
